### PR TITLE
COG-988 Old MFA Verification code field should not be retained in MFA page when user re-logs in after MFA session time out

### PR DESCRIPTION
**Pull request by Nasir2712**
https://osi-cwds.atlassian.net/browse/COG-988

### CODE REVIEW

File: src/components/LoginPage.test.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ describe('LoginPage.js Tests', () => {
     wrapper.setState({mode: MODE.VALIDATING, maskedEmail: 'a@test.com', countDown: countDown})
     wrapper.instance().startTimer()
     expect(wrapper.state().mode).toEqual(MODE.CODE_EXPIRED)
+    expect(wrapper.state().code).toEqual('')
```
Comment (kelfink) [inline src/components/LoginPage.test.js]: doesn't line 126, or somewhere else, have to set the code to something besides '' in order to test this change?

Comment (Nasir2712) [reply src/components/LoginPage.test.js]: no, it needs to be set as blank for it to be empty. The story requires us to keep the MFA code input empty whenever you land on the page.

Comment (kelfink) [reply src/components/LoginPage.test.js]: It just doesn't seem clear that the code wasn't === '' before the startTimer was called. How do we know, unless set setState before the method under test is called ?

Comment (ratneshraval) [reply src/components/LoginPage.test.js]: What @kelfink is saying makes sense. And it works that way too. 
He is suggesting that you set code in line 126 and so we can show that code gets cleared.

Comment (Nasir2712) [reply src/components/LoginPage.test.js]: oh ok, it got it know. I misunderstood.

**Pull request merged**